prov/verbs/src/verbs_eq.c

```diff
@@ -73,14 +73,10 @@ static struct fi_info *
 fi_ibv_eq_cm_getinfo(struct fi_ibv_fabric *fab, struct rdma_cm_event *event,
 		struct fi_info *pep_info)
 {
-	struct fi_info *info, *fi;
+	struct fi_info *info;
 	struct fi_ibv_connreq *connreq;
 
-	fi = fi_ibv_get_verbs_info(ibv_get_device_name(event->id->verbs->device));
-	if (!fi)
-		return NULL;
-
-	info = fi_dupinfo(fi);
+	info = fi_dupinfo(fab->info);
```

Comment:
I mean, the the change in this PR is not correct. Please refer to my first comment. The correction is we should allocate fi_info list (fi_ibv_init_info) and that would increase the setup time.

The current code in the repo caches the fi_info list and avoids this potentially increased setup time.